### DISCUSSION TITLE
How to export the favourite?

### DISCUSSION THREAD

**Discussion by naiximcn** [shedaniel/RoughlyEnoughItems#1668] [Q & A]
I have just updated my minecraft from 1.20.4 to 1.21.

I want to transfer the favourite so that I can continue copy nbt items.

I completely copied my "config" folder in 1.20.4 folder to the 1.21 folder. However, it didn't work any longer!

How can I do this? Help!

**Comment by shedaniel** [comment]
the item representation changed between the versions by mojang, REI doesn’t migrate these data for you since it is not something trivial
